README.md

```diff
@@ -1 +1,29 @@
-# mesh-ui
\ No newline at end of file
+# mesh-ui
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
+```
+
+### Compiles and minifies for production
+```
+npm run build
+```
+
+### Run your end-to-end tests
+```
+npm run test:e2e
+```
+
+### Lints and fixes files
+```
+npm run lint
+```
+
+### Customize configuration
+See [Configuration Reference](https://cli.vuejs.org/config/).
```

Comment:
Did we add a second README with Vue or do these changes apply to the original README?

Reply:
Yeah, the second one over-wrote the original.